### PR TITLE
red dead turtle

### PR DESCRIPTION
**Pull request by SHuang-Broad**
a bit more alarming/brutal.

screen shot 2019-02-25 at 12 16 43 pm

### CODE REVIEW

File: cromshell [hunk 1]
```diff
@@ -88,12 +88,14 @@ function turtle()
 
 function turtleDead()
 {
-  error "    ,,    ,,     ,,       "
-  error "    \\‾\\,-/‾/====/‾/,     "
-  error "     \\/=<‾><‾><‾><‾>-,   "
-  error "     / (\\‾\\‾|‾/‾/‾/‾     "
-  error "    \\‾x/ ˙'-;-;-'˙       "
-  error "     ‾‾                  "
+  export RED='\033[0;31m'
```
Comment (jonn-smith) [inline cromshell]: Can you remove the `RED` and `NC` variables and replace them with the global colors above?

Specifically, `RED` should be `COLOR_FAILED` and `NC` should be `COLOR_NORM`.

Comment (SHuang-Broad) [reply cromshell]: Just updated as you suggested, and if this is what you meant (background color red)

screen shot 2019-02-25 at 12 23 48 pm

screen shot 2019-02-25 at 12 26 03 pm

Comment (jonn-smith) [reply cromshell]: Yup - I figured it would be fine to reuse the color that was already there.

**Comment by jonn-smith** [comment]
Nevermind - I didn't see that there was another divider below.  I'll squash and merge!

**Comment by SHuang-Broad** [comment]
Thanks!

**Pull request merged**